### PR TITLE
[SSHD-799] SSHD port forwarding. Missing SSH_MSG_CHANNEL_OPEN_FAILURE

### PR DESCRIPTION
**Pull request by fcav1973**
message

### CODE REVIEW

File: sshd-core/src/main/java/org/apache/sshd/common/session/helpers/DefaultUnknownChannelReferenceHandler.java [hunk 1]
```diff
@@ -74,8 +74,8 @@ public Channel handleUnknownChannelCommand(
 
             case SshConstants.SSH_MSG_CHANNEL_DATA:
             case SshConstants.SSH_MSG_CHANNEL_EXTENDED_DATA:
-                // Not sure if entirely compliant with RFC4254, but try to stem the flood
-                wantReply = true;
+                // Not sure if entirely compliant with RFC4254, but try to stem the flood	
+                wantReply = false;
                 break;
```
Comment (lgoldstein) [inline sshd-core/src/main/java/org/apache/sshd/common/session/helpers/DefaultUnknownChannelReferenceHandler.java]: This is part of SSHD-800 and should not be part of this patch

**Comment by fcav1973** [comment]
I reverted the commits related to [SSHD-800] and [SSHD-801] for which I openeted https://github.com/apache/mina-sshd/pull/45 PR
